source/common/http/header_utility.cc

```diff
@@ -87,27 +87,29 @@ bool HeaderUtility::matchHeaders(const Http::HeaderMap& request_headers,
   }
 
   bool match;
+  absl::string_view header_view = header->value().getStringView();
```

Comment:
Do we have to make this copy? If so can we TODO removing it?

Reply:
This one is just for readability reasons since it gets pretty busy seeing `header->value().getStringView()` repeated especially when `std::regex_match` takes `begin()` and `end()`. The copy of a string_view should be only slightly worse than a pointer. I've marked it as `const`, which may prevent the copy entirely.

I'm happy to remove it too if you'd like, but I think it makes this much more readable rather than repeating the `header->value().getStringView()`.